spec/factories/factory_bot.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'factory_bot'
+require 'faker'
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
+
+FactoryBot.define do
+  factory :book do
+    genre { Faker::Book.genre }
+    author { Faker::Book.author }
+    image { Faker::File.file_name('foo/bar', 'baz', 'jpg') }
+    editorial { Faker::Book.publisher }
+    year { Faker::Date.birthday }
```

Comment:
Are you sure about this? I think maybe exists a better way with another [Faker::Date](https://github.com/stympy/faker/blob/master/doc/default/date.md)

Reply:
i could use date.backward